spring-5-reactive-client/src/main/java/com/baeldung/reactive/simultaneouswebclient/Client.java

```diff
@@ -0,0 +1,66 @@
+package com.baeldung.reactive.simultaneouswebclient;
+
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+import reactor.core.publisher.Flux;
+import reactor.core.scheduler.Schedulers;
+
```

Comment:
This class needs to be formatted with the eclipse formatter:
https://github.com/eugenp/tutorials/blob/master/eclipse/formatter.xml

Reply:
fixed